### PR TITLE
Add documentation, enhancement and maintenance fields to PR template

### PR DESCRIPTION
**Pull request by Czaki**
# Description
Add new fields to simplify adding labels to PR. 

## Type of change

- [x] Maintenance

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Closes #
 <!-- Please delete options that are not relevant. -->
 - [ ] Bug-fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
+- [ ] Maintenence (changes requuired to napari run smothly)
+- [ ] Enhancement (non-breaking improvements of existing featire)
```
Comment (psobolewskiPhD) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Enhancement (non-breaking improvements of an existing feature)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ Closes #
 <!-- Please delete options that are not relevant. -->
 - [ ] Bug-fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
+- [ ] Maintenence (changes requuired to napari run smothly)
```
Comment (psobolewskiPhD) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Maintenence (changes required to run napari, tests, & CI smoothly)
```

**Review by psobolewskiPhD — APPROVED** [review]

**Comment by psobolewskiPhD** [comment]
I like it—I made some copy fixes in suggestions.

**Comment by Czaki** [comment]
@psobolewskiPhD I have added an additional field. Could you check again?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -17,6 +17,9 @@ Closes #
 <!-- Please delete options that are not relevant. -->
 - [ ] Bug-fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
+- [ ] Maintenence (changes required to run napari, tests, & CI smoothly)
+- [ ] Enhancement (non-breaking improvements of an existing feature)
+- [ ] Documentation (update of docstrings and deprecation informations)
```
Comment (psobolewskiPhD) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Documentation (update of docstrings and deprecation information)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -17,6 +17,9 @@ Closes #
 <!-- Please delete options that are not relevant. -->
 - [ ] Bug-fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
+- [ ] Maintenence (changes required to run napari, tests, & CI smoothly)
```
Comment (jni) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] Maintenance (changes required to run napari, tests, & CI smoothly)
```

**Comment by jni** [comment]
@Czaki I thought we'd decided to do away with the template anyway...?

**Comment by jni** [comment]
(Not that I object to this, it's strictly an improvement towards the local optimum in the template basin that we're in... But we could hop to a different basin and I thought that's what we'd discussed! 😂)

**Comment by Czaki** [comment]
> @Czaki I thought we'd decided to do away with the template anyway...?

when and where? Maybe I miss something? I was even thinking about autolabeler based on pull description.

**Comment by psobolewskiPhD** [comment]
> @Czaki I thought we'd decided to do away with the template anyway...?

Woof, I definitely don't like the idea of *no* template. 
I get that many people are repeat contributors and don't need it or don't care. But they can do Control-A, delete.
But for triaging PRs, it's helpful for there to be some structure, and a good template helps new contributors a lot.

**Comment by jni** [comment]
> when and where? Maybe I miss something?

Can't find it! 😅 It was more limited, about removing the PR checklist. I think @brisvag was part of the conversation but yeah, can't find it. Anyway,

> Woof, I definitely don't like the idea of no template.

This is totally fair. I think better though is to just remove the checklist and make it a simple bullet list, because it's weird that you have to delete *and* tick the box, and you often get things like `- [x ] bugfix` which make me unreasonably upset. 😂 

Either way, all tweaks can come after this PR.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (jni) [reply .github/PULL_REQUEST_TEMPLATE.md]: @Czaki typo fix 👆

**Pull request merged**